app/templates/@layout.latte

```diff
@@ -63,6 +63,7 @@
 		{block main}
 		<div class="main" lang="en">
 			<div class="content has-sidebar">
+				<div class="flash error">See our new modern addons portal <a href="http://bit.ly/cmtte">componette.com</a>. Completely automatic.</div>
```

Comment:
@f3l1x agreed, so I fixed it myself https://github.com/nette/web-addons.nette.org/commit/d4068434ee78a475c59bfac01caf5fcdc95cbff5